svglib/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "svglib" %}
+{% set version = "0.8.1" %}
+{% set hash_type = "sha256" %}
+{% set hash = "1f8cb633f154b83987e3a906a55cf69c8739c0df5ea4eee024677c3e8fb62f55" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - pytest-runner
+    - setuptools
+    - reportlab
+    - lxml
```

Comment:
Yeah every time I have seen `pytest-runner` show up, it seems it needs to be in `build` requirements.